src/main/java/me/jellysquid/mods/sodium/mixin/features/entity/cull/MixinWorldRenderer.java

```diff
@@ -0,0 +1,51 @@
+package me.jellysquid.mods.sodium.mixin.features.entity.cull;
+
+import org.spongepowered.asm.mixin.Final;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+import org.spongepowered.asm.mixin.injection.callback.LocalCapture;
+import me.jellysquid.mods.sodium.client.SodiumClientMod;
+import me.jellysquid.mods.sodium.client.render.SodiumWorldRenderer;
+import net.minecraft.client.MinecraftClient;
+import net.minecraft.client.render.VertexConsumerProvider;
+import net.minecraft.client.render.WorldRenderer;
+import net.minecraft.client.render.entity.EntityRenderDispatcher;
+import net.minecraft.client.util.math.MatrixStack;
+import net.minecraft.entity.Entity;
+
+@Mixin(WorldRenderer.class)
+public class MixinWorldRenderer {
```

Comment:
Can't we just mix this directly in `EntityRenderDispatcher`?

Reply:
Yes that would be much better, thank you for the suggestion. Added with 62a42bf. Injecting into `EntityRenderDispatcher` also allows hitbox, shadow, and fire rendering to be canceled if needed, which was added in 62a42bf.